pin_config.toml

```diff
@@ -0,0 +1,21 @@
+number_of_pins= 4
```

Comment:
Maybe move that file to the already existing config directory, so that all configs are in one place.